packages/dataviews/src/field-types/integer.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal dependencies
+ */
+import type { SortDirection } from '../types';
+
+function sort< Item >(
+	itemA: Item,
+	itemB: Item,
+	direction: SortDirection,
+	field: any
```

Comment:
Instead of "field", I think we may want to just pass `getValue` here to avoid the recursion issues.